builder/main.py

```
@@ -1908,6 +1908,152 @@ def download_fs_action(target, source, env):
+        cmd.extend(shlex.split(extra_args))
+
+    # CLI arguments after --
+    if "--" in sys.argv:
+        dash_index = sys.argv.index("--")
+        if dash_index + 1 < len(sys.argv):
+            cmd.extend(sys.argv[dash_index + 1:])
+
+    cmd.extend(source_files)
+
+    try:
+        result = subprocess.run(cmd, check=False, capture_output=False)
```

Ruff (0.15.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)